packages/app/src/systems/Core/components/UnlockDialog/UnlockDialog.tsx

```diff
@@ -116,7 +120,6 @@ const styles = {
           }
         : {
             width: '350px',
-            height: '600px',
```

Comment:
Why was the height removed, but not the width?

Reply:
because 600px was put initially to match extension popup size, but later we made the fullscreen mode to cover it.

here is with 600px fixed
![image](https://user-images.githubusercontent.com/8636507/213818197-1cf63d10-6d23-4e1b-8ff8-6a0cf5aba2e5.png)

here with no height fixed:
![image](https://user-images.githubusercontent.com/8636507/213818228-79a2d8c2-fd96-4d75-9ae8-65a9d792ca84.png)

we don't need to set fixed height as:
- height will be automatic calculated by content
- we don't need extra space

Reply:
@matt-user FYI I rollback this height change... I'm keeping 600px as it was looking bad after centering app